SDL 0244 - Support for Custom Playback Rate

jacobkeeler opened the pull request:
Implements #221  

This PR is **ready** for review.

### Testing Plan
Manually test with various startTimes, endTimes, and countRates

### Summary
Adds support for countRate parameter in SetMediaClockTimer

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

ShobhitAd commented on app/model/sdl/MediaModel.js:
```diff
@@ -192,21 +195,23 @@ SDL.SDLMediaModel = SDL.ABSAppModel.extend({
 
   setDuration: function() {
 
     var number, str = '', hrs = 0, min = 0, sec = 0;
```
Since this PR is making changes to the existing time calculations it might be a good idea to rename the `number` variable to something to something more descriptive (Maybe `timestamp` or something similar)

jacobkeeler replied:
Fixed in https://github.com/smartdevicelink/sdl_hmi/pull/469/commits/3ed2cdf4a709f502c63728e380fb461730663d6d

jacobkeeler commented:
Addressed comment in https://github.com/smartdevicelink/sdl_hmi/pull/469/commits/3ed2cdf4a709f502c63728e380fb461730663d6d. Also fixed an issue I found where UI.Show would cause the timer to stop
